web-app/src/app/app.component.css

```diff
@@ -24,7 +24,7 @@
 .mat-tab-link {
   line-height: 30px !important;
   height: 3em !important;
-  min-width: 15% !important;
+  min-width: 16.66666%;
```

Comment:
Should the min-width still be !important?

Reply:
ill look into that lol